runit-elixir-focal/1.14/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM socrata/runit-focal AS base
+
+# Set up environment
+ENV LANG C.UTF-8
+
+# Install Java 8. If we start using this image for things other than
+# one app, we might want to revisit this.
+#
+# The update-ca-certs call is to work around a bug in
+# ca-certificates-java that results in missing Java certs
+# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=775775
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
+  DEBIAN_FRONTEND=noninteractive apt-get -o Dpkg::Options::="--force-confnew" --force-yes -fuy install software-properties-common && \
+  DEBIAN_FRONTEND=noninteractive add-apt-repository -y ppa:openjdk-r/ppa && apt-get -y update && \
+  DEBIAN_FRONTEND=noninteractive apt-get -y install openjdk-8-jdk && \
+  mkdir -p /opt && \
+  update-ca-certificates -f
+
+FROM base AS build
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install wget libncurses-dev libssl-dev
+
+RUN mkdir /build /opt/erlang /opt/elixir
+WORKDIR /build
+
+RUN wget https://github.com/erlang/otp/releases/download/OTP-25.3.2.13/otp_src_25.3.2.13.tar.gz
```

Comment:
Looks like the latest is 25.3.2.18 :-)

We almost always want to float to the latest version for compliance reasons.